client/src/views/devices/DevicesPage.vue

```diff
@@ -51,7 +51,7 @@
         <!-- files not downloaded -->
         <div
           class="restore-password"
-          v-if="!passwordSaved"
+          v-if="passwordSaved"
```

Comment:
Nop, it was correct, it should be `v-if="!passwordSaved"`

Reply:
It was just for a preview, a simple oversight.